util/package.js

```diff
@@ -1,5 +1,20 @@
 'use strict';
 
+var allStars = require( 'all-stars' );
+var objectAssign = require( 'object-assign' );
+
+/**
+ * Check all-stars db for given person and assign to person if found.
+ *
+ * @param  {Object} person object representing author or maintainer
+ *
+ * @return {Object}        same object given, possibly modified or augmented
+ */
+function getAllStar( person ) {
+  // override properties from all-stars if available
+  return objectAssign( person, allStars( person ) );
+}
```

Comment:
Ok, I think I can handle that. I will update. :smile_cat: Thanks for working this out with me!